code/src/main/resources/habit_mode/view/codebehind/HabitScreen.fxml

```diff
@@ -64,7 +64,38 @@
                   <TextField fx:id="habitNameTextField" layoutX="22.0" layoutY="80.0" prefHeight="25.0" prefWidth="259.0" promptText="Enter task name..." />
                </children>
             </AnchorPane>
+            <AnchorPane fx:id="removeHabitAnchorPane" layoutX="83.0" layoutY="265.0" prefHeight="219.0" prefWidth="326.0" style="-fx-background-color: white;">
+               <children>
+                  <Label layoutX="8.0" layoutY="14.0" prefHeight="35.0" prefWidth="270.0" text="Remove/Update Habit" textAlignment="CENTER" underline="true">
+                     <font>
+                        <Font size="24.0" />
+                     </font>
+                  </Label>
+                  <Label layoutX="22.0" layoutY="55.0" text="Name:">
+                     <font>
+                        <Font size="16.0" />
+                     </font>
+                  </Label>
+                  <RadioButton fx:id="removeDailyRadioButton" layoutX="22.0" layoutY="144.0" mnemonicParsing="false" selected="true" text="Daily">
+                     <toggleGroup>
+                        <ToggleGroup fx:id="updateFrequencyToggleGroup" />
+                     </toggleGroup>
+                  </RadioButton>
+                  <RadioButton fx:id="removeWeeklyRadioButton" layoutX="121.0" layoutY="144.0" mnemonicParsing="false" text="Weekly" toggleGroup="$updateFrequencyToggleGroup" />
+                  <RadioButton fx:id="removeMonthlyRadioButton" layoutX="215.0" layoutY="144.0" mnemonicParsing="false" text="Monthly" toggleGroup="$updateFrequencyToggleGroup" />
+                  <Button fx:id="updateConfirmButton" layoutX="20.0" layoutY="180.0" mnemonicParsing="false" onAction="#confirmUpdateHabitButtonClicked" text="Confirm" />
+                  <Button fx:id="removeButton" cancelButton="true" layoutX="229.0" layoutY="180.0" mnemonicParsing="false" onAction="#removeButtonClicked" text="Remove" />
+                  <Button fx:id="removeCancelButton" cancelButton="true" layoutX="127.0" layoutY="180.0" mnemonicParsing="false" onAction="#cancelButtonClicked" prefHeight="20.0" prefWidth="66.0" text="Cancel" />
+                  <Label fx:id="updateErrorLabel" focusTraversable="false" layoutX="22.0" layoutY="110.0" text="Please enter a unique task name" textFill="#ee4747" visible="false" />
+                  <TextField fx:id="updateHabitNameTextField" layoutX="22.0" layoutY="80.0" prefHeight="25.0" prefWidth="259.0" promptText="Enter task name..." />
+               </children>
+            </AnchorPane>
          </children>
       </AnchorPane>
+      <Button fx:id="removeHabitsButton" layoutX="435.0" layoutY="144.0" mnemonicParsing="false" onAction="#removeHabitsButtonClicked" prefHeight="53.0" prefWidth="52.0" text="X">
+         <font>
+            <Font size="14.0" />
+         </font>
+      </Button>
```

Comment:
Button renders above the screen-dimming pane. It should be moved up in the file or up on the hierarchy in Scene Builder so that it renders before the screen-dimmer.